files/galaxy/tpv/tools.yml

```diff
@@ -688,6 +688,9 @@ tools:
       - if: input_size >= 25
         fail: Too much data, please check if the input is correct.
 
+  toolshed.g2.bx.psu.edu/repos/devteam/freebayes/freebayes/.*:
+    mem: 9 + input_size * 1
```

Comment:
Should we also set the cores to 10, like we did before?

Reply:
Cores are still set to 10 because they are are inherited from the [shared database](https://github.com/galaxyproject/tpv-shared-database/blob/a8464ae96a13525e26318fde33908563eaebc19c/tools.yml#L411).

See [dry-run](https://github.com/usegalaxy-eu/infrastructure-playbook/actions/runs/6039279480/job/16387559420?pr=881).

```text
Running TPV dry-run for toolshed.g2.bx.psu.edu/repos/devteam/freebayes/freebayes/.*...
!!python/object:galaxy.jobs.JobDestination
converted: false
env:
- {name: GALAXY_MEMORY_MB, value: '9216'}
id: condor_tpv
legacy: false
params: {description: freebayes, docker_memory: 9.0G, metadata_strategy: extended,
  request_cpus: '10', request_memory: 9.0G, submit_request_gpus: '0', tmp_dir: true}
resubmit: []
runner: condor
shell: null
tags: null
url: null
```

Reply:
Puh. This is not intuitive and super hard to see if someone looks that up. I would do it explicitly here or move this entire Def. up to the shared DB

Reply:
Done. Moving it to the shared DB would be very bad, because not every Galaxy instance resubmits held jobs neither they have to do it exactly twice.